test/e2e/performanceprofile/functests/6_mustgather_testing/test_suite_mustgather_test.go

```diff
@@ -1,13 +1,49 @@
 package pao_mustgather
 
 import (
+	"context"
 	"testing"
+	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
+
+	testutils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
+	testclient "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/client"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/junit"
+	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/namespaces"
 )
 
+var _ = BeforeSuite(func() {
+	Expect(testclient.ClientsEnabled).To(BeTrue(), "package client not enabled")
+	// create test namespace
+	err := testclient.Client.Create(context.TODO(), namespaces.TestingNamespace)
```

Comment:
why do we need a testing namespace?

Reply:
It's copied from other suites, not sure if needed.

Reply:
do we have a conclusion? it **seems** to me the namespace is not really needed here